packages/manager/apps/ips/src/components/OptionCard/OptionCard.component.tsx

```diff
@@ -29,8 +41,9 @@ export const OptionCard: React.FC<OptionCardProps> = ({
   const borderStyle = isSelected ? 'option_card_selected' : 'm-[1px]';
   return (
     <OdsCard
+      tabIndex={0}
       className={`flex flex-col p-3 transition-shadow ${stateStyle} ${borderStyle} ${className}`}
-      onClick={() => !isDisabled && onClick?.()}
+      {...handleClick(() => !isDisabled && onClick?.())}
```

Comment:
Question: What is the purpose of using handleClick from MRC here ? Is it to handle click event for key space and enter ?